ouroboros-consensus/test/infra-test/Test/Ouroboros/Consensus/ChainGenerator/Tests/Sync.hs

```diff
@@ -148,87 +145,82 @@ data MockedChainSyncServer m =
     mcssStateQueue :: TQueue m NodeState,
     mcssCurrentState :: StrictTVar m NodeState,
     mcssCandidateFragment :: StrictTVar m TestFragH,
-    mcssUnservedFragment :: StrictTVar m TestFragH,
+    mcssCurrentIntersection :: StrictTVar m (AF.Point TestBlock),
```

Comment:
Documenting the meaning of this field would be helpful. And probably documenting the meaning of all of the mutable fields here would be helpful too.

Reply:
Done in 72b6ec45f although I will probably just decide to rename the field altogether for more clarity.